testsuite/MDAnalysisTests/topology/test_tprparser.py

```diff
@@ -311,26 +438,56 @@ def test_elements():
     tpr_path = TPR
     parser = MDAnalysis.topology.TPRParser.TPRParser(tpr_path)
     topology = parser.parse()
-    reference = np.array((
-         'H,C,H,H,C,H,H,H,C,H,H,H,C,O,N,H,C,H,C,H,H,C,H,C,H,H,H,C,H,H,'
-         'H,C,O,N,H,C,H,H,C,O,O,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H'
-         ',H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,'
-         'O,H,H'
-    ).split(','), dtype=object)
+    reference = np.array(
+        (
+            "H,C,H,H,C,H,H,H,C,H,H,H,C,O,N,H,C,H,C,H,H,C,H,C,H,H,H,C,H,H,"
+            "H,C,O,N,H,C,H,H,C,O,O,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H"
+            ",H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,O,H,H,,"
+            "O,H,H"
```

Comment:
This is a nice trick, could be applied elsewhere. Here just `list("HCHHC")` should work since they are all single-letter elements, but this will work elsewhere.